avaframe/runCom1DFA.py

```diff
@@ -1,30 +1,100 @@
 """
-    Run script for running python DFA kernel
+    Run script for running python com1DFA kernel
 """
+# Load modules
+# importing general python modules
+import time
+import argparse
+import pathlib
+import sys
+
 
 # Local imports
 import avaframe.in3Utils.initializeProject as initProj
 from avaframe.com1DFA import com1DFA
 from avaframe.in3Utils import cfgUtils
 from avaframe.in3Utils import logUtils
+from avaframe.in3Utils import fileHandlerUtils as fU
+
+
+def runCom1DFA(avalancheDir='', calibration='standard'):
+    """ Run com1DFA in the default configuration with only an
+    avalanche directory as input
+
+    Parameters
+    ----------
+    avalancheDir: str
+        path to avalanche directory (setup eg. with init scipts)
+    calibration: str
+        which friction calibration to use: standard, medium, default
+
+    Returns
+    -------
+    peakFilesDF: pandas dataframe
+        dataframe with info about com1DFA peak file locations
+    """
+
+    # Time the whole routine
+    startTime = time.time()
+
+    # log file name; leave empty to use default runLog.log
+    logName = 'runCom1DFA'
+
+    # Load avalanche directory from general configuration file
+    # More information about the configuration can be found here
+    # on the Configuration page in the documentation
+    cfgMain = cfgUtils.getGeneralConfig()
+    if avalancheDir != '':
+        cfgMain['MAIN']['avalancheDir'] = avalancheDir
+    else:
+        avalancheDir = cfgMain['MAIN']['avalancheDir']
+
+    # Start logging
+    log = logUtils.initiateLogger(avalancheDir, logName)
+    log.info('MAIN SCRIPT')
+    log.info('Current avalanche: %s', avalancheDir)
+
+    # ----------------
+    # Clean input directory(ies) of old work and output files
+    # If you just created the ``avalancheDir`` this one should be clean but if you
+    # already did some calculations you might want to clean it::
+    initProj.cleanSingleAvaDir(avalancheDir, deleteOutput=False)
+
+    # Set friction model according to cmd argument
+    cfgCom1DFA = cfgUtils.getModuleConfig(com1DFA, toPrint=False)
+
+    if calibration.lower() == 'small':
+        cfgCom1DFA['GENERAL']['frictModel'] = 'samosATSmall'
+    elif calibration.lower() == 'medium':
+        cfgCom1DFA['GENERAL']['frictModel'] = 'samosATMedium'
+    elif calibration.lower() == 'large':
+        cfgCom1DFA['GENERAL']['frictModel'] = 'samosAT'
+
```

Comment:
shall we add a log.info here if this is done?

Reply:
mh, the log then has the info about the cfg override. This should be enough, right? Because here it requires a manual intervention anyways (via the cmd-line), so the user should be aware of it.